ktor-server/ktor-server-core/src/io/ktor/features/DefaultHeaders.kt

```diff
@@ -15,8 +19,15 @@ class DefaultHeaders(config: Configuration) {
     private var cachedDateTimeStamp: Long = 0L
     @Volatile private var cachedDateText: String = ZonedDateTime.now(ZoneOffset.UTC).toHttpDateString()
 
+    /**
+     * Configuration for [DefaultHeaders] feature.
+     */
     class Configuration {
         val headers = ValuesMapBuilder()
+
+        /**
+         * Adds standard header property [name] with the specified[value]
```

Comment:
Space before `[value]`

Reply:
Also inconsistent use of _period_ at the end of the text, other docs in this PR have them. I wonder what should be the global style on this. Should we end all API docs with period or not?

Reply:
Yes. IMHO, we should use period.